hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/BackgroundPipelineCreator.java

```diff
@@ -32,6 +32,8 @@
 import java.util.concurrent.ScheduledFuture;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.locks.Lock;
+import java.util.concurrent.locks.ReentrantLock;
```

Comment:
Just a kind reminder, since SCM HA is merged back to master, `BackgroundPipelineCreator.java` is deprecated and replaced by `BackgroundPipelineCreatorV2.java`.

Reply:
Thanks @GlenGeng. We are working on bring the SCM HA changes into the upgrade branch, then this PR will be updated. I will move it to a draft for now.